### DISCUSSION TITLE
Can't add secondary account

### DISCUSSION THREAD

**Discussion by adevfz** [community/community#146532] [Discussions]
### Select Topic Area

Bug

### Body

I try to add a secondary account, I login, but the page to verify through mobile is 404. 
Is github okay?
![image](https://github.com/user-attachments/assets/efd26a81-263f-4d68-9f9d-e6c68446cc23)

EDIT:
It worked as soon as I posted this thread, weird. It logged in directly skipping the verification process

**Comment by mecodeatlas** [comment]
Hey, @adevfz! Thanks for reaching out. Great to hear that you managed to add your secondary account. If you encounter any other issues, feel free to respond the thread so other community members can help you :)

**Reply by adevfz** [reply]
I feel like there is still an issue. It should've asked my mobile device for authorization, but I got 404 and after a few retrties it skipped the verification altogether
